packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -154,7 +154,7 @@ export function createOdspNetworkError(
             break;
         default:
             const retryAfterMs = retryAfterSeconds !== undefined ? retryAfterSeconds * 1000 : undefined;
-            error = createGenericNetworkError(errorMessage, true, retryAfterMs, { statusCode });
+            error = createGenericNetworkError(errorMessage, retryAfterMs !== undefined, retryAfterMs, { statusCode });
```

Comment:
This looks wrong to me. It says that if an error is not Throttling error(having retry after x seconds), then we can not retry. We should be able to retry even if time is not defined.

Reply:
Look at impl of `createGenericNetworkError`...  I agree, it's a little wonky.  Just going for a small fix, but maybe it's not the right approach.

Reply:
So this is the impl I can see: 
```
export function createGenericNetworkError(
    errorMessage: string,
    canRetry: boolean,
    retryAfterMs?: number,
    props?: ITelemetryProperties) {
    if (retryAfterMs !== undefined && canRetry) {
        return new ThrottlingError(errorMessage, retryAfterMs / 1000, props);
    }
    return new GenericNetworkError(errorMessage, canRetry, props);
}
```
So after your change all error with no retry time, GenericNetworkError will be non retryable?

Reply:
Yep, this does not look right - default case should be retriable (unless we are ready to examine all the error codes and cases we got here and switch from default case being retryable to default case being non-retryable).